Source/WebKit/GPUProcess/media/RemoteImageDecoderAVFProxy.cpp

```diff
@@ -139,7 +139,7 @@ void RemoteImageDecoderAVFProxy::createFrameImageAtIndex(ImageDecoderIdentifier
         return;
     DestinationColorSpace colorSpace { CGImageGetColorSpace(frameImage.get()) };
 
-    auto bitmap = ShareableBitmap::create(IntSize(width, height), { WTFMove(colorSpace), true /* isOpaque */ });
+    auto bitmap = ShareableBitmap::create(IntSize(width, height), { WTFMove(colorSpace), false /* isOpaque */ });
```

Comment:
I prefer the style of making a local variable "bool isOpaque = false". I think it reads better at the call site.